src/core/tritonserver.cc

```diff
@@ -807,6 +807,10 @@ TRITONSERVER_InferenceTraceNew(
     TRITONSERVER_InferenceTraceReleaseFn_t release_fn, void* trace_userp)
 {
 #ifdef TRITON_ENABLE_TRACING
+  if ((level == TRITONSERVER_TRACE_LEVEL_MIN) ||
```

Comment:
This is the right idea, but level could be (TRITONSERVER_TRACE_LEVEL_TIMESTAMPS | TRITONSERVER_TRACE_LEVEL_TENSORS). So if either min or max is set you need to turn on TIMESTAMPS (level |= TRITONSERVER_TRACE_LEVEL_TIMESTAMPS), and unconditionally you need to mask out MIN and MAX.